fastcrypto/src/groups/ristretto255.rs

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Implementations of the [ristretto255 group](https://ristretto.group/) which is a group of
+//! prime order 2^{252} + 27742317777372353535851937790883648493 built over Curve25519.
```

Comment:
I'm not sure I understand your first point. It this different from the curve25519_dalek_ng imports currently used?

Reply:
I've added a link to the RFC -- good idea! 

Regarding the imports, it is on purpose, that I've used qualified paths for curve25519_dalek_ng's RistrettoPoint, since I wanted to make it clear when we are using fast cryptos RistrettoPoint and when we are referring to curve25519_dalek_ng's. Wouldn't there be a conflict if we import it and also use the same name for a new struct?

Reply:
Fair concern! I think the best approach here is to use the as keyword to alias the import:
https://doc.rust-lang.org/book/ch07-04-bringing-paths-into-scope-with-the-use-keyword.html#providing-new-names-with-the-as-keyword

Reply:
That could work. So something like `use curve25519_dalek_ng::ristretto::RistrettoPoint as ExternalRistrettoPoint`?

Reply:
Exactly!